### DISCUSSION TITLE
Wrong selected style of active nz-tab when embedded in expandable nz-table

### DISCUSSION THREAD

**Discussion by karl-foerster** [NG-ZORRO/ng-zorro-antd#4482] [Q&A] [answered]
### Reproduction link
[https://stackblitz.com/edit/ng-zorro-antd-start-3inmuq](https://stackblitz.com/edit/ng-zorro-antd-start-3inmuq)

### Steps to reproduce
1. Create `<nz-table>` with expand-feature using nzExpand
2. Place a `<nz-tabset>` with tabs within this expandable area

### What is expected?
The active tab of the tabset within the expandable area should have correct style with blue title and blue line under the title.

### What is actually happening?
The tab title is just colored blue but has no blue line under the title. Instead the style is applied after active tab change when clicking one of the tabs.

| Environment | Info |
|---|---|
| ng-zorro-antd | 8.5.1 |
| Browser | All Browsers |

**Comment by marlag** [comment]
Same if nested tabset, see https://stackblitz.com/edit/angular-426xym-9x8waa - select tab 2. Would it be commonly fixed or another issue needed?

@worki2k1 Programmatically dispatched window resize works as workaround (at least for nested tabs).

**Comment by karl-foerster** [comment]
@marlag Seems to work as workaround.

**Comment by hsuanxyz** [comment, marked as answer]
```diff
<tr [nzExpand]="data.expand">
  <td colspan="3">
-    <nz-tabset>
+    <nz-tabset *ngIf="data.expand">
      <nz-tab nzTitle="Tab 1">
        Content of Tab 1
      </nz-tab>
      <nz-tab nzTitle="Tab 2">
        Content of Tab 2
      </nz-tab>
    </nz-tabset>
  </td>
</tr>
```
